cassandra/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gcr.io/google_containers/cassandra:v5
+MAINTAINER Dana NicCaluim <dniccaluim@airware.com>
+
+RUN apt-get update
+RUN apt-get install -y procps psmisc
+
+ADD https://raw.githubusercontent.com/openzipkin/zipkin/master/zipkin-cassandra-core/src/main/resources/cassandra-schema-cql3.txt /schema.txt
```

Comment:
probably want to use a variable to ensure a stable tag is used as opposed to `master`. ex `1.9.0`